source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,20 @@
 ============
 Introduction
 ============
+
+oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
+ from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.
+
+Library consists of the following major components: Data Management, Algorithms, and Services
+
+The Data Management component includes classes and utilities for data acquisition, initial preprocessing
+ and normalization, for data conversion into numeric formats done by one of supported Data Sources,
+ and for model representation.
+
+The Algorithms component consists of classes that implement algorithms for data analysis (data mining),
+ and data modeling (training and prediction). These algorithms include matrix decompositions, clustering,
+ classification, and regression algorithms, as well as association rules discovery.
+
+The Services component includes classes and utilities used across Data Management and Algorithms components.
+ These classes enable memory allocation, error handling, and implementation of collections and shared
```

Comment:
Remove collections/shared pointers for now. We have to decide, if they would be part of the public api

Reply:
not fixed

Reply:
fixed